src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -409,7 +409,7 @@ public void MessageBox(string message)
 
         public void VerifyDialog(string dialogAutomationId, bool isOpen)
         {
-            var dialogAutomationElement = FindDialog(dialogAutomationId, isOpen);
+            var dialogAutomationElement = DialogHelpers.FindDialog(GetDTE().MainWindow.HWnd, dialogAutomationId, isOpen);
```

Comment:
Should FindDialog take something else than the HWND so we're not having to repeat the GetDTE().MainWindow.HWnd thing every time? I admit I'm not sure when I'd ever want to write something else.